### PR TITLE
Best borer

### PR DESCRIPTION
**Pull request by Gef21**
Adds 1/10,000 chance of a borer egg hatching into a super cool special borer with new emotes, a new look (made by blithering), a new walk (made by blithering) and a new smell

http://i.imgur.com/1lF9a0U.png
http://i.imgur.com/BiV3qbY.png

🆑 

- rscadd: Cool new borer sprites by blithering and a super secret suprise

### CODE REVIEW

**Comment by Gef21** [comment]
I realised my mistake don't meme me for it

**Comment by ghost** [comment]
My immersions

File: code/modules/mob/living/simple_animal/borer/egg.dm [hunk 1]
```diff
@@ -68,13 +68,23 @@
...
 	if(O)
-		var/turf/T = get_turf(src)
-		src.visible_message("<span class='notice'>\The [name] bursts open!</span>")
-		var/mob/living/simple_animal/borer/B = new (T, child_prefix_index)
-		B.transfer_personality(O.client)
-		// Play hatching noise here.
-		playsound(src.loc, 'sound/items/borer_hatch.ogg', 50, 1)
-		qdel(src)
+		var defect = rand(1,10000)
+		if(defect == 666)
```
Comment (adriandias) [inline code/modules/mob/living/simple_animal/borer/egg.dm]: :1st_place_medal:

File: code/modules/mob/living/simple_animal/borer/egg.dm [hunk 2]
```diff
@@ -68,13 +68,23 @@
...
 	var/mob/dead/observer/O = args["player"]
 	if(O)
-		var/turf/T = get_turf(src)
-		src.visible_message("<span class='notice'>\The [name] bursts open!</span>")
-		var/mob/living/simple_animal/borer/B = new (T, child_prefix_index)
-		B.transfer_personality(O.client)
-		// Play hatching noise here.
-		playsound(src.loc, 'sound/items/borer_hatch.ogg', 50, 1)
-		qdel(src)
+		var defect = rand(1,10000)
```
Comment (PJB3005) [inline code/modules/mob/living/simple_animal/borer/egg.dm]: You should use `var/defect` here.

File: code/modules/mob/living/simple_animal/borer/egg.dm (hunk 1 above)
Comment (PJB3005) [inline code/modules/mob/living/simple_animal/borer/egg.dm]: You shouldn't copy paste all this code, and make a variable that gets set to the type of borer to spawn, then use that variable and remove the copy paste.

**Review by PJB3005 — APPROVED** [review]

**Review by PJB3005 — CHANGES_REQUESTED** [review]
Wrong button

File: code/modules/mob/living/simple_animal/borer/egg.dm (hunk 1 above)
Comment (Gef21) [reply code/modules/mob/living/simple_animal/borer/egg.dm]: I wanted to do that but honestly I don't really know what I'm doing

File: code/modules/mob/living/simple_animal/borer/egg.dm (hunk 2 above)
Comment (Gef21) [reply code/modules/mob/living/simple_animal/borer/egg.dm]: done

**Comment by Aurx** [comment]
That sprite looks like trash.

**Comment by Aurx** [comment]
Disregard above, this entire thing is just trash.  
What in the world is the intended point of this, aside from shoving shit into the repo?

**Comment by yclatious** [comment]
Literally just a joke pr aurx, don't get mad, its gef we are talking about here, you should not expect a serious pr from him.

**Comment by Gef21** [comment]
it adds depth to borers

File: code/modules/mob/living/simple_animal/borer/egg.dm [hunk 3]
```diff
@@ -68,10 +68,15 @@
 /obj/item/weapon/reagent_containers/food/snacks/borer_egg/proc/recruiter_recruited(var/list/args)
 	var/mob/dead/observer/O = args["player"]
 	if(O)
+		var/defect = rand(1,10000)
 		var/turf/T = get_turf(src)
 		src.visible_message("<span class='notice'>\The [name] bursts open!</span>")
-		var/mob/living/simple_animal/borer/B = new (T, child_prefix_index)
-		B.transfer_personality(O.client)
+		if(defect == 666)
```
Comment (PJB3005) [inline code/modules/mob/living/simple_animal/borer/egg.dm]: I meant you should do the following:
```dm
var/borer_type = (defect == 666 ? /mob/living/simple_animal/borer/defected_borer : /mob/living/simple_animal/borer)

var/mob/living/simple_animal/borer/B = new borer_type(T, child_prefix_index)
B.transfer_personality(O.client)
```

**Review by PJB3005 — CHANGES_REQUESTED** [review]

**Review by PJB3005 — APPROVED** [review]

**Comment by SangriaMonkey** [comment]
:eggplant:

**Comment by Megamaw** [comment]
Finally, I can play a dog.

**Comment by SonixApache** [comment]
delete this

**Comment by Probe1** [comment]
If it EVER comes up and people hate it we'll revert it.

**Comment by Intigracy** [comment]
I hate you.  I also suck dicks for coke.

**Comment by Exxion** [comment]
This is excellent except the word is "defective"

**Pull request merged**